ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -485,23 +485,18 @@ open class PillButtonBar: UIScrollView, TokenizedControlInternal {
         }
     }
 
-    private func updatePillButtonTokens() {
+    private func updateAppearance() {
```

Comment:
Can this be renamed updatePillButtonAppearance? It's only changing the buttons, not the bar.

Reply:
Fascinating. There's no API for updating the bar itself, huh? Seems like a pretty significant oversight. And fixing that's out of the scope of this PR, so I'll just do the rename for right now.